src/webgpu/api/operation/rendering/draw.spec.ts

```diff
@@ -1,15 +1,27 @@
 export const description = `
 Tests for the general aspects of draw/drawIndexed/drawIndirect/drawIndexedIndirect.
 
-TODO: plan and implement
-- draws (note bind group state is not tested here):
-    - various zero-sized draws
-    - draws with vertexCount not aligned to primitive topology (line-list or triangle-list) (should not error)
-    - index buffer is {unset, set}
-    - vertex buffers are {unset, set} (some that the pipeline uses, some it doesn't)
-      (note: to test this, the shader in the pipeline doesn't have to actually use inputs)
-    - x= {draw, drawIndexed, drawIndirect, drawIndexedIndirect}
-- ?
+TODO:
+* basic - Test that basic draw functionality works
```

Comment:
Is this all one test (one test function)? If so can you briefly describe here how it'll generate a testable draw call?

Reply:
Done, but also split it a bit.